### ISSUE TITLE
WARNING: Using / for division outside of calc() is deprecated and will be removed in Dart Sass 2.0.0

### ISSUE THREAD

**Issue by lotoras** [bootstrap-vue/bootstrap-vue#6952]
### Describe the bug

When compiling sass file I have few deprecation warnings
WARNING: Using / for division outside of calc() is deprecated and will be removed in Dart Sass 2.0.0.
Recommendation: math.div($spacer, 2) or calc($spacer / 2)
More info and automated migrator: https://sass-lang.com/d/slash-div

### Steps to reproduce the bug

Compile scss using Dart-SASS

### Versions

**Libraries:**

- BootstrapVue: 2.21.2
- Bootstrap: 4.6.1
- Sass: 1.32.13
- Vue: 2.6.14

**Comment by Hiws** [comment]
Should be fixed in the latest release.

**Comment by vekinox** [comment]
This is not fixed in 2.23.1 (using sass-loader 13.2.0)

**Comment by VividLemon** [comment]
> This is not fixed in 2.23.1 (using sass-loader 13.2.0)

You're missing some context, such as bootstrap version

**Comment by mattnelsonuk** [comment]
As above, version 2.23.1 still has some issues.

    "bootstrap": "4.6.2",
    "bootstrap-vue": "2.23.1"


```
   ╷             
18 │     top: ($font-size-lg * $line-height-lg - $b-custom-control-indicator-size-lg) / 2;
   │          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   ╵
    node_modules/bootstrap-vue/src/components/form-checkbox/_form-checkbox.scss 18:10  @import
    node_modules/bootstrap-vue/src/components/form-checkbox/index.scss 1:9             @import
    node_modules/bootstrap-vue/src/components/index.scss 5:9                           @import
    node_modules/bootstrap-vue/src/index.scss 14:9                                     @import
    src/@core/scss/core.scss 19:9                                                      root stylesheet

Deprecation Warning: Using / for division outside of calc() is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($font-size-sm * $line-height-sm - $b-custom-control-indicator-size-sm, 2) or calc(($font-size-sm * $line-height-sm - $b-custom-control-indicator-size-sm) / 2)

More info and automated migrator: https://sass-lang.com/d/slash-div
```
